requirements.txt

```diff
@@ -5,3 +5,5 @@ Shapely==1.7.1
 tqdm==4.60.0
 dataclasses==0.8
 pyshp==2.1.3
+sqlalchemy
```

Comment:
Out of interest, why do you need sqlalchemy?

Reply:
It is what I use to connect to the database where the indices are held (sqlite now, postgresql as soon as we move that way). Once the database is up, I can go ahead and process the data into month data/vehicle

Reply:
I should point that the Python code is temporary, as it reads the daily files...

Reply:
OK.